Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            const int personCount = 2;
+
+            for (int i = 0; i < personCount; i++)
+            {
+                ProcessBmi();
+            }
+
+            Console.ReadKey();
+        }
+
+        public static void ProcessBmi()
+        {
+            string firstName = PromptString("Input your first name: ");
+            string lastName = PromptString("Input your last name: ");
+            int age = PromptInt("Input your age: ");
+            float height = PromptFloat("Input your height in m: ");
+            float weight = PromptFloat("Input your weight in kg: ");
+            float bmi = CalculateBmi(weight, height);
+
+            if (Math.Abs(bmi - (-1)) < 0.01)
+            {
+                return;
+            }
+            Console.WriteLine($"{firstName} {lastName} is {age} years old, with a weight of {weight} kg and a height of {height} cm.\nCalculated BMI is {bmi}");
+            Console.WriteLine();
+        }
+        public static int PromptInt(string message)
+        {
+            Console.Write(message);
+
+            if (!int.TryParse(Console.ReadLine(), out int result))
+            {
+                return -1;
+            }
+
+            if (result <= 0)
+            {
+                return 0;
+            }
+
+            return result;
+        }
+
+        public static float PromptFloat(string message)
+        {
+            Console.Write(message);
+
+            var style = NumberStyles.Float;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'style'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5130208)